.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,9 +13,7 @@ Please provide as much information as you can.
 - [ ] .NET Jupyter Notebook 
 
 #### What kind of error was it?
```

Comment:
I think we can do away with this heading now.

Reply:
Agreed.